spec/features/landing_page_spec.rb

```diff
@@ -29,4 +29,16 @@
     expect(page).to have_css "img[src='#{district.logo.url}']"
     expect(page).to_not have_content 'DISTRICT FOO'
   end
+
+  scenario 'submission made through contact form' do
+    visit root_path
+    expect(page).to have_content 'CONTACT US'
+
+    fill_in 'Name', with: 'David'
+    fill_in 'Email', with: 'schoolbotty@test.com'
```

Comment:
Use `example.com` for test email addresses – it's guaranteed not to go anywhere if we somehow accidentally send a real email.